tasks/discover-granules/package.json

```diff
@@ -1,53 +1,47 @@
 {
-  "name": "@cumulus/discover-granules",
-  "version": "1.13.0",
-  "description": "Discover Granules in FTP/HTTP/HTTPS/SFTP/S3 endpoints",
-  "main": "index.js",
-  "directories": {
-    "test": "tests"
-  },
-  "homepage": "https://github.com/nasa/cumulus/tree/master/tasks/discover-granules",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/nasa/cumulus"
-  },
-  "engines": {
-    "node": ">=8.10.0"
-  },
-  "scripts": {
+	"name": "@cumulus/discover-granules",
```

Comment:
The changes in this file are just in the tab size? Is that intentional? I'd prefer to keep it consistent

Reply:
That was unintentional, not sure why that happened. Fixed now.